phoenix-core/src/it/java/org/apache/phoenix/end2end/BackwardCompatibilityIT.java

```diff
@@ -86,33 +66,12 @@
 @Category(NeedsOwnMiniClusterTest.class)
 public class BackwardCompatibilityIT {
 
-    private static final String SQL_DIR = "sql_files/";
-    private static final String RESULTS_AND_GOLD_FILES_DIR = "gold_files/";
-    private static final String COMPATIBLE_CLIENTS_JSON = 
-            "compatible_client_versions.json";
-    private static final String BASH = "/bin/bash";
-    private static final String EXECUTE_QUERY_SH = "scripts/execute_query.sh";
-    private static final String QUERY_PREFIX = "query_";
-    private static final String RESULT_PREFIX = "result_";
-    private static final String GOLD_PREFIX = "gold_";
-    private static final String SQL_EXTENSION = ".sql";
-    private static final String TEXT_EXTENSION = ".txt";
-    private static final String CREATE_ADD = "create_add";
-    private static final String CREATE_DIVERGED_VIEW = "create_diverged_view";
-    private static final String ADD_DATA = "add_data";
-    private static final String ADD_DELETE = "add_delete";
-    private static final String QUERY_CREATE_ADD = QUERY_PREFIX + CREATE_ADD;
-    private static final String QUERY_ADD_DATA = QUERY_PREFIX + ADD_DATA;
-    private static final String QUERY_ADD_DELETE = QUERY_PREFIX + ADD_DELETE;
-    private static final String QUERY_CREATE_DIVERGED_VIEW = QUERY_PREFIX + CREATE_DIVERGED_VIEW;
-    private static final String MVN_HOME = "maven.home";
-    private static final String JAVA_TMP_DIR = "java.io.tmpdir";
+    protected final String compatibleClientVersion;
+    protected static Configuration conf;
+    protected static HBaseTestingUtility hbaseTestUtil;
+    protected static String zkQuorum;
+    protected static String url;
```

Comment:
nit: Make package-protected instead. Also, some of these diffs may not be needed if you're just going to use the `doSetup()` of SplitSystemCatalogIT